test/Suites/Plutus/Model/Script/V2/Test/Oracle/Hashed.hs

```diff
@@ -139,7 +139,7 @@ victoryBy :: Fraud -> App -> PubKeyHash -> PubKeyHash -> Run ()
 victoryBy Fraud{..} app@App{..} user oracle =
   withUtxo (hasOracleDatum . snd) oracle $ \(oracleRef, _) -> do
     betBox <- getBet app
-    withDatum @AnswerDatum oracleRef $ \oracleDatum ->
+    withDatum @_ @AnswerDatum oracleRef $ \oracleDatum ->
```

Comment:
Use Monad parameter as second argument in this function and everywhere else
to avoid @_. As script type changes more often than monad type